spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/FeignClientsConfiguration.java

```diff
@@ -56,6 +58,9 @@
 	@Autowired(required = false)
 	private List<AnnotatedParameterProcessor> parameterProcessors = new ArrayList<>();
 
+	@Autowired(required = false)
+	private List<FeignCustomizer> feignCustomizers = new ArrayList<>();
```

Comment:
I kind of left it open-ended in case we later decided there were other extension points that could also be handled by a custom type, like `WebMvcConfigurer`. But whatever you guys are comfortable with is fine with me. @dsyer ?